phys2bids/phys2bids.py

```diff
@@ -262,6 +262,7 @@ def phys2bids(filename, info=False, indir='.', outdir='.', heur_file=None,
     # Run analysis on trigger channel to get first timepoint and the time offset.
     # #!# Get option of no trigger! (which is wrong practice or Respiract)
     phys_in.check_trigger_amount(chtrig, thr, num_timepoints_expected, tr)
+    thr = phys_in.thr
```

Comment:
I would delete this line and use straight away `phys_in.thr` where needed (if I saw it right, it's just for the trigger plot, ~line 278)

Reply:
I disagree, if the trigger is not inputed then we use the thr from the function, with this line we avoid using an if statement in case the user inputs a trigger